### PR TITLE
OSD-19863: Lower retention for management cluster CMO prometheus data

### PR DESCRIPTION
**Pull request by typeid**
### What type of PR is this?
bug

### What this PR does / why we need it?

This PR lowers the retention of prometheus data for management clusters to avoid running into full PVs. 

### Which Jira/Github issue(s) this PR fixes?

https://issues.redhat.com/browse/OSD-19863

### Pre-checks (if applicable):
- [ ] Tested latest changes against a cluster
- [ ] Included documentation changes with PR
- [ ] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

File: deploy/cluster-monitoring-config-non-uwm/config.yaml
Comment (mjlshen) [inline deploy/cluster-monitoring-config-non-uwm/config.yaml:1]: I think https://github.com/openshift/managed-cluster-config/blob/master/deploy/cluster-monitoring-config/config.yaml also needs to be excluded from landing on MC's

File: deploy/cluster-monitoring-config-non-uwm/config.yaml [hunk 1]
```diff
@@ -1,10 +1,12 @@
 deploymentMode: "SelectorSyncSet"
 selectorSyncSet:
-  matchLabelsApplyMode: "OR"
```
Comment (typeid) [inline deploy/cluster-monitoring-config-non-uwm/config.yaml:3]: I moved the 4.5 specific out of here so we don't have the `matchLabelsApplyMode: "OR"`

**Review by NautiluX — COMMENTED** [review]
/lgtm

**Pull request merged**